sitebuilder/app/controllers/api/ItemsController.php

```
@@ -281,8 +287,13 @@ public function create()
 		$item->set($this->request->data);
 		$item->site_id = $this->site()->id;
 		$item->type = $category->type;
+		$itemCreationService = new ItemCreation();
 
-		if ($item->save()) {
+		list ($created, $errors) = $itemCreationService->create($item, [
```

You'd probably want to return the errors in the API. If I'm trying to create an item, and only get a generic error message back, I won't be very likely to find out what's wrong.